src/Identity/UI/src/Areas/Identity/Pages/V4/Account/ConfirmEmailChange.cshtml

```diff
@@ -4,5 +4,5 @@
     ViewData["Title"] = "Confirm email change";
 }
 
-<h2>@ViewData["Title"]</h2>
+<h1>@ViewData["Title"]</h1>
```

Comment:
@HaoK this likely changes the visuals, are PMs ok with the new visual aspect?

We can use the `h2` and so on Bootstrap classes to keep the visual aspect the same while being correct about the markup.